Tests/Kernels/ConsoleTest/Kernel.cs

```diff
@@ -32,100 +32,119 @@ protected override void Run()
         {
             try
             {
-                Console.Clear();
```

Comment:
Please rework the Console tests. I think we should run a series of tests for each resolution, similar to how its done in Graphics Test.